### PR TITLE
Mvp 276 comtrade data

### PR DESCRIPTION
**Pull request by depsiatwal**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if updating requirements) Requirements have been compiled.

### CODE REVIEW

File: dataservices/helpers.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+import pandas as pd
+import datetime
+
+
+class ComTradeData():
```
Comment (richtier) [inline dataservices/helpers.py]: no need for paren

File: dataservices/helpers.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    url = (
+        'https://comtrade.un.org/api/get?type=C&freq=A&px=HS&rg=1'
```
Comment (richtier) [inline dataservices/helpers.py]: can be single line

File: dataservices/helpers.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+import pandas as pd
```
Comment (richtier) [inline dataservices/helpers.py]: why renaming as `pd`? bit harder to read. I thought "what's pd?" and had to scroll up to find out. If it was not renamed then devs will not have to do that :)

File: dataservices/helpers.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+class ComTradeData():
+    url = (
+        'https://comtrade.un.org/api/get?type=C&freq=A&px=HS&rg=1'
+    )
+
+    def __init__(self, commodity_code, reporting_area, partner_country='United Kingdom'):
+        pd.set_option('mode.chained_assignment', None)
+        self.product_code = self.get_product_code(commodity_code)
+        self.reporting_area_id = self.__get_comtrade_company_id(reporting_area)
```
Comment (richtier) [inline dataservices/helpers.py]: private methods make unit testing that method hard, and make reading a bit harder. It also makes devs wonder "why is this private". Overall net negative when it comes to maintainability.

File: dataservices/helpers.py [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        self.partner_country_id = self.__get_comtrade_company_id(partner_country)
+
+    def __get_comtrade_company_id(self, country_name):
+        with open('dataservices/resources/reporterAreas.json', 'r') as f:
+            json1_str = f.read()
+            counties_data = pd.DataFrame(json.loads(json1_str)['results'])
+            if not counties_data[counties_data['text'] == country_name]['id'].empty:
```
Comment (richtier) [inline dataservices/helpers.py]: double negative. instead of `if not a == b` it's easier to read `if a != b`

Comment (richtier) [inline dataservices/helpers.py]: what's happening here?

`counties_data[counties_data['text'] == country_name]`

that evaluates to `counties_data[True]` or `counties_data[False]`

Comment (depsiatwal) [reply dataservices/helpers.py]: No this is used for filtering it actually returns a pandas dataseries of true/false values which is then used to filter the dataframe. 

Make sense ?

I could break up the line if it doesn't

**Pull request merged**